.github/workflows/test.yaml

```diff
@@ -5,5 +5,7 @@ on:
 
 jobs:
   unit-tests:
-    uses: canonical/operator-workflows/.github/workflows/test.yaml@8892eb826818585b397295e40276ddd0c5d3d459
-    secrets: inherit
```

Comment:
Yeah, the `inherit` option fails because of a name mismatch. The referenced workflow looks expects `GITHUB_TOKEN`, but in the repo it's named `GH_TOKEN`. So, we needed the mapping.

Reply:
It turns out it didn't solve the issue. So I reverted to the `inherit` option.